docs/zh/latest/discovery/kubernetes.md

```diff
@@ -0,0 +1,106 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# 基于 Kubernetes 的服务发现
+
+Kubernetes 服务发现插件以 ListWatch 方式监听 Kubernetes 集群的 的 v1.endpoints 的实时变化,
+并将其值存储在 ngx.shared.DICT 中, 同时遵循 APISix Discovery 规范提供对外查询接口
+
+# Kubernetes 服务发现插件的配置
+
+Kubernetes 服务发现的样例配置如下:
+
+```yaml
+discovery:
+  kubernetes:
+    service:
+      # kubernetes apiserver schema, options [ http | https ]
+      schema: https #default https
+
+      # kubernetes apiserver host, options [ ipv4 | ipv6 | domain | env variable]
+      host: 10.0.8.95 #default ${KUBERNETES_SERVICE_HOST}
```

Comment:
I suggest writing kubernetes.default directly here, what do you think?

Reply:
In cluster

Reply:
@zhixiongdu027 what do you think?

Reply:
In the cluster we can get the address of the `kubernetes` service through `KUBERNETES_SERVICE_HOST`, but once we really need to verify the SSL certificate, most of the time we do not issue a certificate for the IP address. Domain names are far more generic than IP addresses.

Reply:
| most of the time we do not issue a certificate for the IP address. Domain names are far more generic than IP addresses.

Usually this is the case.

But in the clusters I have seen, the certificates used by the apiserver all contain ${KUBERNETES_SERVICE_HOST} and Host_IP,

So even if "ssl_verify=true "
It is no problem to use ${KUBERNETES_SERVICE_HOST} as the server address within the Pod.

Reply:
We can keep the status  for now.